specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -12806,6 +15566,90 @@
         }
       }
     },
+    "ContinuousWebJob": {
+      "description": "Continuous Web Job Information.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "ContinuousWebJob resource specific properties",
+          "properties": {
+            "status": {
+              "description": "Job status.",
+              "type": "string"
+            },
+            "detailedStatus": {
+              "description": "Detailed status.",
+              "type": "string"
+            },
+            "logUrl": {
+              "description": "Log URL.",
+              "type": "string"
+            },
+            "name": {
```

Comment:
Artifact of the ARM free KUDU world.
@ Eric, annotate the property with [ApiExplorerSettings(IgnoreApi = true)] in model and regenerate the swagger.

---
In reply to: [134406400](https://github.com/Azure/azure-rest-api-specs/pull/1551#discussion_r134406400) [](ancestors = 134406400)